Examples/Java/Sources/Board.java

```diff
@@ -0,0 +1,56 @@
+//
+// Board.java
+// Autogenerated by plank
+//
+// DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+// @generated
+//
+
+package com.pinterest.models;
+
+import android.support.annotation.IntDef;
+import com.google.auto.value.AutoValue;
+import java.net.URI;
+import java.util.Set;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringDef;
+import java.lang.annotation.Retention;
+import java.util.Date;
+import java.util.List;
+import java.lang.annotation.RetentionPolicy;
+import java.util.Map;
+
+@AutoValue
+public abstract class Board {
+
+
+    public abstract @SerializedName("id") @Nullable String identifier();
+    public abstract @SerializedName("contributors") @Nullable Set<User> contributors();
+    public abstract @SerializedName("counts") @Nullable Map<String, Integer> counts();
+    public abstract @SerializedName("created_at") @Nullable Date createdAt();
+    public abstract @SerializedName("creator") @Nullable Map<String, String> creator();
+    public abstract @SerializedName("description") @Nullable String descriptionText();
+    public abstract @SerializedName("image") Image image();
```

Comment:
Is there a way to make Image generate in a different package and get the import correct for it?

I'm thinking of feature package that make like a `MyCoolHomeFeedType` that uses a `User` type from a different package.

Reply:
Not for a single-invocation of Plank but we can look into this in a follow-up task. Currently to achieve that you would need to have multiple calls to plank and avoid creating models for referenced schemas and be explicit about which models you wanted to create with which package.